app/waf/src/controllers/adc.controller.ts

```diff
@@ -37,15 +37,16 @@ export class AdcController {
       },
     },
   })
-  async create(@requestBody() adc: Partial<Adc>): Promise<Adc> {
+  async create(@requestBody() adc: Partial<Adc>): Promise<AdcResponse> {
     if (!adc.id) {
       adc.id = uuid();
     }
 
     try {
-      return await this.adcRepository.create(adc);
+      let data = await this.adcRepository.create(adc);
```

Comment:
The `data` variable is only used once after the declaration/assignment. 
I'm thinking that if we need to remove it, thus: 

(not check whether or not can run)
```
return new AdcResponse(
    await this.adcRepository.create(adc)
);

```
Only one statement...

Reply:
It's not right or wrong, but only concern about "Code of Conduct" or standard.

Reply:
Done